becquerel/core/utils.py

```diff
@@ -3,12 +3,25 @@
 from __future__ import print_function
 import datetime
 from dateutil.parser import parse as dateutil_parse
-from six import string_types
 from uncertainties import UFloat, unumpy
 import numpy as np
 
 VECTOR_TYPES = (list, tuple, np.ndarray)
 
+try:
+
+    isinstance("", basestring)
+
+    def isstring(s):
+        """Test for strings in python2 and 3"""
+        return isinstance(s, basestring)
+
+except NameError:
+
+    def isstring(s):
```

Comment:
Can you add comments as to which is for `py2` and `py3`?

Reply:
Yeah good point too. I will do so.

Reply:
@cosama just a reminder about this

Reply:
The version is mentioned in the docstring now.